src/main/java/com/internet/shop/storeservice/impl/OrderServiceImpl.java

```diff
@@ -0,0 +1,54 @@
+package com.internet.shop.storeservice.impl;
+
+import com.internet.shop.dao.OrderDao;
+import com.internet.shop.library.Inject;
+import com.internet.shop.library.Service;
+import com.internet.shop.models.Order;
+import com.internet.shop.models.ShoppingCart;
+import com.internet.shop.storeservice.OrderService;
+import com.internet.shop.storeservice.ShoppingCartService;
+import java.util.List;
+
+@Service
+public class OrderServiceImpl implements OrderService {
+    @Inject
+    private OrderDao orderDao;
+
+    @Inject
+    private ShoppingCartService shoppingCartService;
+
+    @Override
+    public Order completeOrder(ShoppingCart shoppingCart) {
+        Order order = new Order(shoppingCart.getUserId());
+        order.setProducts(List.copyOf(shoppingCart.getAllProducts()));
+        orderDao.create(order);
+        shoppingCartService.clear(shoppingCart);
+        return order;
+    }
+
+    @Override
+    public Order get(Long id) {
+        return orderDao.getById(id)
+                .orElseThrow();
```

Comment:
тут краще відразу викликати get() адже якщо всереди немає ніякого елемента  - він по дефолту викине NoSuchElement()
аналогічно з рештою випадків
```suggestion
        return orderDao.getById(id).get();
```